src/main/java/services/FileReaderService.java

```diff
@@ -0,0 +1,7 @@
+package services;
+
+import java.util.List;
+
+public interface FileReaderService {
+    List<String> getFruitData(String filePath);
+}
```

Comment:
```suggestion
public interface FileReaderService {
    List<String> read(String filePath);
}
```

as we can use this method to read any files with any data